Challange-1-iOS/Challange-1-iOS/Network/NetworkManager.swift

```diff
@@ -52,4 +55,31 @@ class NetworkManager {
 
         task.resume()
     }
+
+    func rxExecuteNetworkCall<ResultType: Decodable>(_ call: APIProtocol) -> Single<ResultType> {
+            let decoder = JSONDecoder()
```

Comment:
It seems there are some extra spaces in the indentation here. Can you check?

Reply:
done